packages/tooling/cfp-template/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@microsoft/cfp-template",
+  "description": "The default FAST project template",
+  "version": "0.1.0",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/fast/issues"
+  },
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config=./template/webpack.prod.js",
+    "eslint": "eslint . --ext .ts",
+    "eslint:fix": "eslint . --ext .ts --fix",
+    "start": "webpack-dev-server --config=./template/webpack.dev.js --history-api-fallback --progress",
+    "test": "npm run eslint && npm run test:e2e",
+    "serve": "node ./template/server.js",
+    "test:e2e": "npm run build && npx playwright test --config=./template/playwright.config.js"
```

Comment:
I wonder if we should have two different test commands, one for unit tests and one for e2e tests and then a command to run them both. Looks like we only have the e2e tests right now. It should be possible to handle both with Playwright. See the link to modern-web.dev below that shows Playwright unit testing with Chai. Not sure how much is required to set this up from scratch without that site's library though. Worth looking into I think.

Reply:
When/if we get consensus from others on the testing strategy I'll go ahead and implement this change, just waiting on responses.